recipes/xtensor/run_test.bat

```diff
@@ -0,0 +1,6 @@
+cd %SRC_DIR%\test
+conda env create -f .\test-environment.yml
+activate test-xtensor
```

Comment:
Missed this before. This is not a great solution. The main problems I see is this environment is not getting cleaned up and an existing environment may be removed by using this strategy.

Would be better if we fixed issue ( https://github.com/conda/conda-build/issues/1059 ).